packages/components/pages/f-checkout/test-utils/component-objects/f-checkout.component.js

```diff
@@ -282,6 +307,12 @@ module.exports = class Checkout extends Page {
         this.fields.tableIdentifier.input.setValue(customerInfo.tableIdentifier);
     }
 
+    populateAgeVerificationForm (dob) {
```

Comment:
Minor thing but you _could_ destructure the values out here?

```suggestion
    populateAgeVerificationForm ({ day, month, year }) {
```